server/app/views/applicant/TrustedIntermediaryDashboardView.java

```diff
@@ -95,14 +104,16 @@ private FormTag renderSearchForm(
                 .url())
         .with(
             FieldWithLabel.input()
+                .setId("search")
                 .setFieldName("search")
-                .setValue(search.orElse(""))
-                .setLabelText("Search")
+                .setValue(searchParameters.search().orElse(""))
+                .setLabelText("SearchText")
```

Comment:
Blocking: I'd add a space here since this is user facing. Also, perhaps it should say something like "Search by Name"